FogCreek/Glitch-Community
# Fortunate archeology: componentize Report Abuse popover

## Links
* Remix link: https://fortunate-archeology.glitch.me/

## GIF/Screenshots:
![Screen Shot 2019-05-21 at 4 48 55 PM](https://user-images.githubusercontent.com/938722/58129852-b755e680-7be8-11e9-9f6c-4d6ca3c6ab53.png)
![Screen Shot 2019-05-21 at 4 27 19 PM](https://user-images.githubusercontent.com/938722/58129864-ba50d700-7be8-11e9-8957-9037419aa5a9.png)
![Screen Shot 2019-05-21 at 4 24 34 PM](https://user-images.githubusercontent.com/938722/58129870-bd4bc780-7be8-11e9-9e76-694536c0765f.png)

## Changes:
* add topLeft & topRight to popover alignments
* style refinements on PopoverInfo, PopoverTitle, InfoDescription
* componentize ReportAbusePop

## How To Test:
- visit https://fortunate-archeology.glitch.me/
  * clicking Report Abuse should open the Report Abuse window
  * your email address should be validated as you type
  * the report should be validated as you type
  * submitting a report should create a ticket in `#abuse`
 - log in with your persistent user token
  * the Report Abuse flow should be the same, except it shows your username instead of a field for an email address
